index.html

```diff
@@ -393,6 +393,13 @@
  <section>
    <h2>Web Payment Operations</h2>
 
+     <div class="issue" title="Registration should be loosley coupled to payment">
```

Comment:
Please convert this to an issue in the issue tracker and use the data-number and title fields to link to it? If you don't have time to do that, I'll just pull it in and do it after the pull.